### PR TITLE
Changes from Process CG call 

### PR DESCRIPTION
**Pull request by wseltzer**
See discussion in https://www.w3.org/2018/08/15-w3process-minutes.html
must request for class 3; must secure for class 4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,7 +35,8 @@
 </style> <!--[if lt IE 9]><script src='undefined://www.w3.org/2008/site/js/html5shiv.js'></script><![endif]-->
   </head>
   <body>
-    <div class="head"><a href="https://www.w3.org/"><img alt="W3C" src="https://www.w3.org/Icons/w3c_home" height="48" width="72"></a>
+    <div 
+	 ="head"><a href="https://www.w3.org/"><img alt="W3C" src="https://www.w3.org/Icons/w3c_home" height="48" width="72"></a>
```
Comment (frivoal) [inline index.html]: this part of the diff looks like a typo: accidental line break, dropped `class`.

**Comment by wseltzer** [comment]
https://github.com/w3c/w3process/issues/67

File: index.html [hunk 2]
```diff
@@ -2107,7 +2108,7 @@ <h4 id="correction-classes">6.2.5 Classes of Changes</h4>
       </dl>
 	    <h4 id="contributor-license">6.2.X Contributor License Grants</h4>
 	    <p>When a contributor who is not a Working Group participant offers a change in class 3 or 4 (as described in <a href="correction-classes">6.2.5</a>)
-		other than an error correction, the Team shall secure from the contributor a recorded royalty-free patent commitment.
+		the Team shall request from the contributor a recorded royalty-free patent commitment; for a change in class 4, the Team shall secure such commitment.
```
Comment (frivoal) [inline index.html]: LGTM

Comment (dwsinger) [reply index.html]: LGTM

**Comment by frivoal** [comment]
The substantive change that was requested has been fixed, so that's good, but the markup-breaking typo is still there.

**Comment by chaals** [comment]
Hmm. I would prefer for simplicity that we use MUST and do not use SHALL.

But whatevs.

**Comment by dwsinger** [comment]
chaals is right, we use IETF conformance terms not ISO (must, not shall). can you tweak?

**Comment by vfournier17** [comment]
I still have a question about what the written license agreement will look like.  It should mirror

**Comment by frivoal** [comment]
> chaals is right, we use IETF conformance terms not ISO (must, not shall). can you tweak?

done (https://github.com/w3c/w3process/commit/864c9741a2db074a0e367a64fa19e69c6c80b1f6)

**Pull request merged**